db/other_sql/clean_case_number_v01.sql

```diff
@@ -0,0 +1,39 @@
+-- replace crf_number with any case number field
```

Comment:
I'm not sure where this is used, but just so you know, clean case number already exists on both `doc_sentences` and `ocso_warrants` as generated columns. See these files:

- `20230621153902_add_clean_case_number_to_ocso_warrants`
- `20240710130156_add_clean_case_number_to_doc_sentences`

If I remember correctly, both SQL statements could be improved a bit